HomeHealthServices/R4/files/HomeHealthServicesPlanOfCarePrepopulation-0.0.1.cql

```diff
@@ -0,0 +1,125 @@
+library HomeHealthServicesPlanOfCarePrepopulation version '0.0.1'
+using FHIR version '4.0.0'
+include FHIRHelpers version '4.0.0' called FHIRHelpers
+
+codesystem "ICD-10-CM": 'http://hl7.org/fhir/sid/icd-10-cm'
```

Comment:
Are these Code Systems used in this CQL?

Reply:
Good catch. Removed.